gatsby-node.js

```diff
@@ -76,7 +81,28 @@ exports.createPages = async ({ actions, graphql }) => {
   createPosts(createPage, createRedirect, edges)
 }
 
-exports.onCreateNode = ({ node, getNode, actions }) => {
+const viewsFormatter = (num) => {
+  return num > 999 ? `${(num / 1000).toFixed(1)}k` : `${num}`
```

Comment:
Use `Intl.NumberFormat` for a generic logic for number formating (please mention me if you need a hand)